src/Dumplie/Domain/Customer/Products.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\Customer;
+
+use Dumplie\Domain\Customer\Exception\ProductNotFoundException;
+
+interface Products
+{
+    /**
+     * @param string $sku
+     *
+     * @throws ProductNotFoundException
+     *
+     * @return Product
+     */
+    public function getBySku(string $sku) : Product;
```

Comment:
I would probably use VO here, just to make sure that invariants of SKU is protected